### PR TITLE
man: update Spanish translation

### PR DESCRIPTION
**Pull request by ikks**
man: update Spanish translation

- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/man.md [hunk 1]
```diff
@@ -3,30 +3,34 @@
 > Da formato y muestra páginas del manual.
 > Más información: <https://manned.org/man>.
 
-- Muestra la página del manual para un comando:
+- Muestra la página del manual de un comando:
 
 `man {{comando}}`
 
-- Muestra la página del manual para un comando de la sección 7:
+- Abra la página del manual de un comando en un navegador(requiere que la variable `BROWSER` esté establecida):
```
Comment (kant) [inline pages.es/linux/man.md:10]:
```suggestion
- Abre la página del manual de un comando en un navegador (requiere que la variable `BROWSER` esté establecida):
```

File: pages.es/linux/man.md [hunk 2]
```diff
@@ -3,30 +3,34 @@
...
+- Muestra las rutas usadas en la búsqueda de las páginas:
 
 `man --path`
 
 - Muestra la ubicación de la página del manual en lugar de la propia página:
 
 `man --where {{comando}}`
 
-- Muestra la página del manual usando una ubicación específica:
+- Muestra la página del manual usando un idioma(locale) específico:
```
Comment (kant) [inline pages.es/linux/man.md:30]:
```suggestion
- Muestra la página del manual usando un idioma específico:
```

File: pages.es/linux/man.md [hunk 3]
```diff
@@ -3,30 +3,34 @@
...
 
 `man --locale={{locale}} {{comando}}`
```
Comment (kant) [inline pages.es/linux/man.md:32]:
```suggestion
`man --locale={{idioma}} {{comando}}`
```

**Review by kant — APPROVED** [review]

**Comment by ikks** [comment]
Screenshot from 2024-10-24 10-57-20
Seems tldr-bot has a false positive here too. Thanks for the feedback 😺

**Comment by kant** [comment]
> Screenshot from 2024-10-24 10-57-20 Seems tldr-bot has a false positive here too. Thanks for the feedback 😺

check line 32

**Comment by ikks** [comment]
Please take a look at line 30, which explains idioma, better to leave it as locale?
> 
> check line 32

**Review by kant — APPROVED** [review]

File: pages.es/linux/man.md [hunk 4]
```diff
@@ -3,30 +3,34 @@
...
 
 - Muestra la ubicación de la página del manual en lugar de la propia página:
 
 `man --where {{comando}}`
 
-- Muestra la página del manual usando una ubicación específica:
+- Muestra la página del manual usando un idioma (locale) específico (p.e. es para español):
 
-`man --locale={{locale}} {{comando}}`
+`man --locale={{idioma}} {{comando}}`
```
Comment (kant) [inline pages.es/linux/man.md:32]:
```suggestion
`man --locale {{idioma}} {{comando}}`
```

**Comment by ikks** [comment]
@sebastiaanspeck I'm convinced we are synced, even if the bot complains. Please help me here too
```
i@h:tldr$ pr -t -w 140 -m pages/linux/man.md pages.es/linux/man.md
# man								     # man
								     
> Format and display manual pages.				     > Da formato y muestra páginas del manual.
> More information: <https://manned.org/man>.			     > Más información: <https://manned.org/man>.
								     
- Display the man page for a command:				     - Muestra la página del manual de un comando:
								     
`man {{command}}`						     `man {{comando}}`
								     
- Open the man page for a command in a browser (requires the `BROWSER - Abre la página del manual de un comando en un navegador (requiere qu
								     
`man --html {{command}}`					     `man --html {{command}}`
								     
- Display the man page for a command from section 7:		     - Muestra la página del manual de la sección 7 de un comando:
								     
`man {{7}} {{command}}`						     `man {{7}} {{comando}}`
								     
- List all available sections for a command:			     - Lista todas las secciones disponibles para un comando:
								     
`man --whatis {{command}}`					     `man --whatis {{comando}}`
								     
- Display the path searched for manpages:			     - Muestra las rutas usadas en la búsqueda de las páginas:
								     
`man --path`							     `man --path`
								     
- Display the location of a manpage rather than the manpage itself:   - Muestra la ubicación de la página del manual en lugar de la propia pág
								     
`man --where {{command}}`					     `man --where {{comando}}`
								     
- Display the man page using a specific locale:			     - Muestra la página del manual usando un idioma (locale) específico (p.
								     
`man --locale {{locale}} {{command}}`				     `man --locale={{idioma}} {{comando}}`
								     
- Search for manpages containing a search string:		     - Busca las páginas del manual que contienen la cadena de búsqueda:
								     
`man --apropos "{{search_string}}"`				     `man --apropos "{{cadena_de_búsqueda}}"`
```

**Comment by kant** [comment]
I can not see the whole strings. Please do a capture as usual you did in the past. Thanks

**Comment by ikks** [comment]
Sure, thank you @kant 
Screenshot from 2024-10-25 12-16-15
Screenshot from 2024-10-25 12-18-04

File: pages.es/linux/man.md (hunk 4 above)
Comment (ikks) [reply pages.es/linux/man.md]: I just reviewed, sorry about that, the = somehow went through.  Just fixed it. Thank you again.

**Pull request merged**